mev-boost-rs/Cargo.toml

```diff
@@ -23,6 +23,8 @@ ethereum-consensus = { git = "https://github.com/ralexstokes/ethereum-consensus"
 beacon-api-client = { git = "https://github.com/ralexstokes/beacon-api-client" }
 
 mev-rs = { path = "../mev-rs" }
+lazy_static = "1.4.0"
```

Comment:
how much of a hassle is it to use `OnceCell` or `OnceLock`?

just stabilized in latest rust stable: https://blog.rust-lang.org/2023/06/01/Rust-1.70.0.html

Reply:
good look! I'll see how a refactor with that would go